astrology.lic

```diff
@@ -219,7 +218,10 @@ class Astrology
 
     if @have_telescope
       bput('get telescope', 'you get', 'You are already', "That can't be picked up")
-      bput("center telescope on #{body}", 'You put your eye')
+      if bput("center telescope on #{body}", 'You put your eye', 'open it to make any use of it').match('open it')
```

Comment:
Rubocop warns me about this line, should I use =~ instead? I dont see any issues, beyond style